.github/workflows/build.yml

```diff
@@ -55,10 +95,15 @@ jobs:
         run: |
           TAG=${{ steps.tag-images.outputs.tag }}
           docker login --username discoursebuild --password $DOCKERHUB_PASSWORD
-          docker push discourse/base:2.0.$TAG-slim
-          docker push discourse/base:slim
-          docker push discourse/base:2.0.$TAG
-          docker push discourse/base:release
+          docker push discourse/base:2.0.$TAG-slim${{ steps.arch-helper.outputs.arch_postfix_dash }}
+          docker push discourse/base:slim${{ steps.arch-helper.outputs.arch_postfix_dash }}
+          docker push discourse/base:2.0.$TAG${{ steps.arch-helper.outputs.arch_postfix_dash }}
+          docker push discourse/base:release${{ steps.arch-helper.outputs.arch_postfix_dash }}
+      - name: Push discourse/base:aarch64 image for backwards compatibility
+        if: success() && (github.ref == 'refs/heads/main') && (matrix.arch == 'arm64')
+        run: |
+          docker tag discourse/base:release${{ steps.arch-helper.outputs.arch_postfix_dash }} discourse/base:aarch64
+          docker push discourse/base:aarch64
```

Comment:
This is for backwards compatibility since we have been publishing `discourse/base:aarch64`. The plan is to eventually publish `discourse/base:release` as a manifest so we can think about deprecating this then.